lib/matplotlib/patches.py

```diff
@@ -181,14 +181,30 @@ def contains(self, mouseevent, radius=None):
 
     def contains_point(self, point, radius=None):
         """
-        Returns *True* if the given point is inside the path
+        Returns `True` if the given *point* is inside the path
         (transformed with its transform attribute).
```

Comment:
True should be double left quotes, "``" here and below....

Reply:
Actually, by double left quotes do you mean \`\`True\`\` or \`True\`? 

It was \`True\` already, and now I've updated them to \`\`True\`\` but feel that I should confirm this...

Reply:
The updated version is what I meant.  But your tests are still broken, and it looks like you may have to rebase on master to resolve the conflict in `test_patches.py`.  If you need help w/ that, let us know; I think it only took my 10 tries until I figured out how to do it properly (basically rebase all your commits *first* and then rebase onto master...)

Reply:
Yes, aware of the test failing but I couldn't figure out why...

Reply:
The error messages are in the logs of the tests.  Just need to click through.

If you need to debug on your local machine (reccomended), then have a look at https://matplotlib.org/devdocs/devel/testing.html  If anything is unclear in there let us know!